cmp/tab/TabContainerModel.js

```diff
@@ -269,6 +274,22 @@ export class TabContainerModel extends HoistModel {
         if (nextTab) this.activateTab(nextTab.id);
     }
 
+    //-----------------------------
+    // Trigger load / refreshes of TabContainer directly.
+    // Note that we are intentionally *not* implementing LoadSupport here.
+    //-----------------------------
+    async loadAsync(loadSpec) {
```

Comment:
Don't fully understand this change, and would like to discuss, but it does seem to me that you should only need to override loadAsync and everything else should follow.